bus/routes.go

```diff
@@ -2158,6 +2160,13 @@ func (b *Bus) multipartHandlerCreatePOST(jc jape.Context) {
 		key = object.GenerateEncryptionKey(object.EncryptionKeyTypeSalted)
 	}
 
+	if req.MimeType == "" {
```

Comment:
Wouldn't it be better to add this on the download side? Or both? That way we'll fix existing setups too.

Reply:
I'd rather set the mime type once on upload rather than inferring it every time to make sure it stays the same once the file is uploaded.
The fix to the UI issue will be the `dl` flag which prevents accidentally streaming the files in the first place.